packages/angular/common/src/directives/navigation/router-outlet.ts

```diff
@@ -296,7 +297,8 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
 
       // Calling `markForCheck` to make sure we will run the change detection when the
       // `RouterOutlet` is inside a `ChangeDetectionStrategy.OnPush` component.
-      enteringView = this.stackCtrl.createView(this.activated, activatedRoute);
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      enteringView = this.stackCtrl.createView(this.activated!, activatedRoute);
```

Comment:
The build started to fail because `this.activated` was potentially `null`, but `createView` does not accept `null` for this param. This has always been a problem, but it only started to fail with Angular 16.